profiles/common/modules/custom/tmgmt_poetry/inc/tmgmt_poetry.webservice.inc

```diff
@@ -101,7 +112,12 @@ function _tmgmt_poetry_authenticate($user, $password) {
  * Function available to call from our webservice.
  */
 function FPFISPoetryIntegrationRequest($user, $password, $msg) {
-  watchdog('Poetry: Response', filter_xss(htmlentities($msg)), array(), WATCHDOG_DEBUG);
+  watchdog(
+    'tmgmt_poetry',
+    "Receive request: !msg",
```

Comment:
t() is applied on the watchdog messages when outputting them, e.g. in theme_dblog_message(), see https://api.drupal.org/api/drupal/modules!dblog!dblog.admin.inc/function/theme_dblog_message/7.x. @voidtek is right it should not be used as you suggested @tatarbj.